src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -3074,7 +3082,7 @@ static void vdo_resume(struct dm_target *ti)
 static struct target_type vdo_target_bio = {
 	.features = DM_TARGET_SINGLETON,
 	.name = "vdo",
-	.version = { 9, 0, 0 },
+	.version = { 9, 0, 1 },
 #ifdef __KERNEL__
```

Comment:
I feel like this iis more of a y-version change, so move it to 9.1.0 instead of 9.0.1. Any particular reason for this choice?

Reply:
Not particularly I guess. This seems a minor change so I made it 9.0.1

Reply:
Fixed to 9.1.0

Reply:
the 9.1.0 is more of a version that lets clients know when features are changed/added. In this case, we are saying that you can now call this new dmsetup message. LVM uses this to decide when to do various actions with VDO. I still think having a version for the config also makes sense. If we change the format, we don't need to update the module version (this is not something we should be changing often) and just use the format version instead.